### ISSUE TITLE
Inference output prediction

### ISSUE THREAD

**Issue by N-NEJATISHAHIDIN** [ratschlab/HIRID-ICU-Benchmark#15]
Hi,
If we want to evaluate the accuracy in classification task, how we can convert the predicted tensor to the predicted label? 
For example in Mortality task-classification setup, If the predicted output is [-2.4, 1.1] how we can compute the final predicted class? Can we simply assume a max operation? Or do we need to multiply classes with some defined wights?

**Comment by hugoych** [comment]
Hi Negar,
Thanks a lot for using our work! 
 
The outputs of our models are logits and not probabilities, following `pytorch` best practice.  The reason is to be compatible with loss functions that apply `LogSoftmax` to the input of the function. 

Thus to obtain a probability, you need to apply a [softmax](https://pytorch.org/docs/stable/generated/torch.nn.Softmax.html) to the prediction along the last dimension. This is what de in practice, when defining `self.output_transform`  [here](https://github.com/ratschlab/HIRID-ICU-Benchmark/blob/9207412cb894b2b40244c6605022ddf309935f95/icu_benchmarks/models/wrappers.py#L60) and applying it at evaluation [here](https://github.com/ratschlab/HIRID-ICU-Benchmark/blob/9207412cb894b2b40244c6605022ddf309935f95/icu_benchmarks/models/wrappers.py#L246). 

Hope this answers your question. If so, please close this issue. If not, happy to continue the discussion!
